examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -3006,6 +3015,15 @@ cluster DoorLock = 257 {
   attribute access(write: administer) optional boolean sendPINOverTheAir = 50;
   attribute access(write: administer) optional boolean requirePINforRemoteOperation = 51;
   attribute access(write: administer) optional int16u expiringUserTimeout = 53;
+  readonly attribute access(read: administer) optional nullable octet_string<65> aliroReaderVerificationKey = 128;
+  readonly attribute access(read: administer) optional nullable octet_string<16> aliroReaderGroupIdentifier = 129;
+  readonly attribute access(read: administer) optional octet_string<16> aliroReaderGroupSubIdentifier = 130;
+  readonly attribute access(read: administer) optional octet_string<16> aliroExpeditedTransactionSupportedProtocolVersions = 131;
```

Comment:
AliroExpeditedTransactionSupportedProtocolVersions is a `list[octstr]` in my spec copy and it seems unchanged in the update. this version seems to say it is a non-list.

Reply:
good catch. i ll add the entryType

Reply:
done.